### ISSUE TITLE
gradle build fails

### ISSUE THREAD

**Issue by danielkroeni** [Frege/FregeFX#3]
The build fails on my machine. After a fresh clone of the current version (4facd204909d1242955a1983667c6ce73d9dfaea) I get:

cortex:FregeFX dk$ ./gradlew run
FAILURE: Build failed with an exception.
- What went wrong:
  A problem occurred configuring root project 'fregefx'.
  > Could not resolve all dependencies for configuration ':classpath'.
  > Could not find org.frege-lang:frege-gradle-plugin:0.4.2-SNAPSHOT.
   Searched in the following locations:
       file:/Users/dk/.m2/repository/org/frege-lang/frege-gradle-plugin/0.4.2-SNAPSHOT/maven-metadata.xml
       file:/Users/dk/.m2/repository/org/frege-lang/frege-gradle-plugin/0.4.2-SNAPSHOT/frege-gradle-plugin-0.4.2-SNAPSHOT.pom
       file:/Users/dk/.m2/repository/org/frege-lang/frege-gradle-plugin/0.4.2-SNAPSHOT/frege-gradle-plugin-0.4.2-SNAPSHOT.jar
       https://repo1.maven.org/maven2/org/frege-lang/frege-gradle-plugin/0.4.2-SNAPSHOT/maven-metadata.xml
       https://repo1.maven.org/maven2/org/frege-lang/frege-gradle-plugin/0.4.2-SNAPSHOT/frege-gradle-plugin-0.4.2-SNAPSHOT.pom
       https://repo1.maven.org/maven2/org/frege-lang/frege-gradle-plugin/0.4.2-SNAPSHOT/frege-gradle-plugin-0.4.2-SNAPSHOT.jar
   Required by:
       org.frege-lang:fregefx:0.2.0-SNAPSHOT

Since I have no local installation of the plugin I tried activating this part:
plugins {
  id "org.frege-lang" version "0.4.3"
}

cortex:FregeFX dk$ ./gradlew run
FAILURE: Build failed with an exception.
- Where:
  Build file '/Users/dk/Desktop/FregeFX/build.gradle' line: 14
- What went wrong:
  An exception occurred applying plugin request [id: 'org.frege-lang', version: '0.4.3']
  > Failed to apply plugin [id 'org.frege-lang']
  > Could not find property 'classes' on task set.

When I try the snippets from https://plugins.gradle.org/plugin/org.frege-lang I again get a different error:
cortex:FregeFX dk$ ./gradlew run
FAILURE: Build failed with an exception.
- Where:
  Build file '/Users/dk/Desktop/FregeFX/build.gradle' line: 36
- What went wrong:
  A problem occurred evaluating root project 'fregefx'.
  > Could not find method frege() for arguments [org.frege-lang:frege:3.23.288-gaa3af0c] on org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler_Decorated@78d71df1.

I would like to build and run the code from https://github.com/Frege/FregeFX/blob/master/examples/src/main/frege/fregefx/example/HelloWorld.fr within my own eclipse project and for the moment I would be happy if I could just copy a bunch of jars into my project.

Thanks 
Dani

**Comment by Dierk** [comment]
I pushed the latest changes to master, please try again with
    gradlew install

**Comment by danielkroeni** [comment]
It worked for me after removing javafx.scene.layout.CornerRadiiConverter as described in https://github.com/Frege/FregeFX/issues/2.

My Java Version:

```
cortex:fregeTest dk$ java -version
java version "1.8.0_60"
Java(TM) SE Runtime Environment (build 1.8.0_60-b27)
Java HotSpot(TM) 64-Bit Server VM (build 25.60-b23, mixed mode)
```

I'm not sure what this CornerRadiiConverter is. But in 1.8.0_60 they moved it to `com.sun.javafx.scene.layout.region.CornerRadiiConverter`.

Thanks

**Comment by danielkroeni** [comment]
This problem is fixed with commit e4803085e3688cf9d3975fe6483680f572bd0aa7
Thanks Dierk
